scipy/sparse/linalg/_eigen/_svds.py

```diff
@@ -290,21 +285,21 @@ def svds(A, k=6, ncv=None, tol=0, which='LM', v0=None,
 
     The next example follows that of 'sklearn.decomposition.TruncatedSVD'.
 
-    >>> rng = np.random.RandomState(0)
+    >>> rng = np.random.default_rng(0)
     >>> X_dense = rng.random(size=(100, 100))
     >>> X_dense[:, 2 * np.arange(50)] = 0
     >>> X = sparse.csr_array(X_dense)
-    >>> _, singular_values, _ = svds(X, k=5, random_state=rng)
+    >>> _, singular_values, _ = svds(X, k=5, rng=rng)
     >>> print(singular_values)
-    [ 4.3293...  4.4491...  4.5420...  4.5987... 35.2410...]
+    [ 4.3221...  4.4043...  4.4907...  4.5858... 35.4549...]
 
     The function can be called without the transpose of the input matrix
     ever explicitly constructed.
 
     >>> rng = np.random.default_rng(102524723947864966825913730119128190974)
     >>> G = sparse.random_array((8, 9), density=0.5, random_state=rng)
```

Comment:
Can we change this one, too?
```suggestion
    >>> G = sparse.random_array((8, 9), density=0.5, rng=rng)
```

Reply:
Oh, right. Doing too many of these at once!